numberformat/proposed.emu

```diff
@@ -691,7 +691,7 @@
       <h1>FormatNumericToString ( _intlObject_, _x_ )</h1>
 
       <p>
-        The FormatNumericToString abstract operation is called with arguments _intlObject_ (which must be an object with [[RoundingType]], [[MinimumSignificantDigits]], [[MaximumSignificantDigits]], [[MinimumIntegerDigits]], [[MinimumFractionDigits]], [[MaximumFractionDigits]], [[RoundingIncrement]], and [[TrailingZeroDisplay]] internal slots), and _x_ (which must be a mathematical value or ~negative-zero~), and returns a Record containing two values: _x_ as a String value with digits formatted according to the five formatting parameters in the field [[FormattedString]], and the final mathematical value (or *-0*<sub>𝔽</sub>) of _x_ after rounding has been performed in the field [[RoundedNumber]].
+        The FormatNumericToString abstract operation is called with arguments _intlObject_ (which must be an object with [[RoundingType]], [[MinimumSignificantDigits]], [[MaximumSignificantDigits]], [[MinimumIntegerDigits]], [[MinimumFractionDigits]], [[MaximumFractionDigits]], [[RoundingIncrement]], and [[TrailingZeroDisplay]] internal slots), and _x_ (which must be a mathematical value or ~negative-zero~), and returns a Record containing two values: _x_ as a String value with digits formatted according to the five formatting parameters in the field [[FormattedString]], and the final Intl mathematical value of _x_ after rounding has been performed in the field [[RoundedNumber]].
       </p>
 
       <emu-alg>
```

Comment:
I added parts of this to the PR. I didn't add the whole things because:

1. Some of these changes touch lines unrelated to the proposal (and as of yet unchanged in the proposal), and I'd like to keep the overall proposal's diff on the smaller side.
2. For `_missingDigitCount_`, I'm not a huge fan of performing the subtraction before the size comparison. It's likely safer for implementations to perform the comparison first, since subtraction is subject to overflow. Probably not a huge deal in this particular case, but it's generally a good pattern and I see no reason to change it.